### PR TITLE
Rewrite of compose documentation

### PR DESCRIPTION
**Pull request by sgallagher**
Since we have now migrated away from ODCS and to using Pungi directly on compose hosts, this patch updates the documentation to reflect that and to provide details on how to manually start a compose and/or forcibly sync a new compose to the mirrors if needed.

NOTE: These changes assume that https://pagure.io/pungi-fedora/pull-request/1397 will be accepted.

### CODE REVIEW

File: modules/ROOT/pages/deliverables.adoc [hunk 1]
```diff
@@ -5,20 +5,18 @@ WARNING: Fedora ELN is a development environment. It is not intended for product
...
-
-A mirror of this compose is also available at http://mirror.facebook.net/fedora-eln/production/latest-Fedora-ELN/
+The latest production compose of Fedora ELN packages is available at https://download.fedoraproject.org/pub/eln/1/
 
 See also xref:compose.adoc[Compose].
 
 == Container image
 
-As a part of the compose, we build also container images which are then published to Quay.io image registry under the name `quay.io/fedoraci/fedora:eln`.
+As a part of the compose, we build also container images which are then published to Quay.io image registry under the name `quay.io/fedoraci/eln`.
```
[the remark this answers is not in the document]
Comment (sgallagher) [reply modules/ROOT/pages/deliverables.adoc:14]: The change is intentional, as we're now being delivered through the official Fedora and Quay registries. But you are correct about the backticks; I will fix it.

File: modules/ROOT/pages/deliverables.adoc [hunk 2]
```diff
@@ -5,20 +5,18 @@ WARNING: Fedora ELN is a development environment. It is not intended for product
 
 == Compose
 
-The latest production compose of Fedora ELN packages is available at https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/
-
-A mirror of this compose is also available at http://mirror.facebook.net/fedora-eln/production/latest-Fedora-ELN/
+The latest production compose of Fedora ELN packages is available at https://download.fedoraproject.org/pub/eln/1/
```
Comment (Conan-Kudo) [inline modules/ROOT/pages/deliverables.adoc:8]: This is odd, why isn't this the ELN version?

File: modules/ROOT/pages/deliverables.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (sgallagher) [reply modules/ROOT/pages/deliverables.adoc:14]: Oh, whoops. I see the mistake now on a second read. Will fix that too...

**Comment by sgallagher** [comment]
Review changes made.

**Review by Conan-Kudo — APPROVED** [review]

**Pull request merged**